src/desktop/index.js

```diff
@@ -25,10 +25,7 @@ app.use(require("./apps/artwork"))
 app.use(require("./apps/artwork2/server"))
 app.use(require("./apps/about"))
 app.use(require("./apps/collect_art"))
-if (sd.NEW_COLLECT_PAGE_ENABLED) {
-  app.use(require("./apps/collect2/server"))
-}
-app.use(require("./apps/collect"))
+app.use(require("./apps/collect2/server"))
```

Comment:
Should we remove the old collect app and rename the new one to collect?

Reply:
I thought about it but left there just in case something in the new one goes horribly wrong and we want to revert the change.

Reply:
Makes sense.